libs/sdk-ui/api/sdk-ui.api.md

```diff
@@ -1684,7 +1692,7 @@ export function useInsightDataView(config: IUseInsightDataViewConfig & UseInsigh
 export type UseInsightDataViewCallbacks = UseCancelablePromiseCallbacks<DataViewFacade, GoodDataSdkError>;
 
 // @public
-export function usePagedResource<TParams, TItem>(resourceFactory: (params: TParams) => Promise<IPagedResource<TItem>>, fetchParams: TParams[], fetchDeps: React.DependencyList, resetDeps: React.DependencyList, preventResetPromises?: boolean, getCacheKey?: (params: TParams) => string, initialState?: IUsePagedResourceState<TItem>): IUsePagedResourceResult<TItem>;
+export function usePagedResource<TParams, TItem>(resourceFactory: (params: TParams) => Promise<IPagedResource<TItem>>, fetchParams: TParams[], fetchDeps: React.DependencyList, resetDeps: React.DependencyList, ignorePromiseCacheActions?: IIgnorePromiseCacheActions, getCacheKey?: (params: TParams) => string, initialState?: IUsePagedResourceState<TItem>): IUsePagedResourceResult<TItem>;
```

Comment:
This is a breaking change on public interface

Reply:
This param was just introduced three weeks ago in this PR: https://github.com/gooddata/gooddata-ui-sdk/pull/3967/files#diff-ee802a48022d7961551e87ddb3f3851a75adb9c639256fcd1254b63abeb3efe1R1687. Only one place used this param is in `libs/sdk-ui-dashboard/src/presentation/insightList/InsightList.tsx` line 128. So, I think it won't break many places.

Reply:
Well, then even that recent change is breaking

Reply:
Hi @kandl, I have updated the code to move it as the last parameter of that function. Please help to review again! Thank you!